src/Raven.Server/Documents/DocumentIdWorker.cs

```diff
@@ -56,8 +56,15 @@ public static ByteStringContext<ByteStringMemoryCache>.InternalScope GetSliceFro
             }
         }
 
-        public static ByteStringContext<ByteStringMemoryCache>.InternalScope GetSliceFromId<TTransaction>(TransactionOperationContext<TTransaction> context, ReadOnlySpan<char> id, out Slice idSlice, byte? separator = null)
+        [MethodImpl(MethodImplOptions.AggressiveInlining)]
+        public static ByteStringContext<ByteStringMemoryCache>.InternalScope GetSliceFromId<TTransaction>(TransactionOperationContext<TTransaction> context,
+            ReadOnlySpan<char> id, out Slice idSlice, byte? separator = null)
             where TTransaction : RavenTransaction
+        {
+            return GetSliceFromId(context.Allocator, id, out idSlice, separator);
+        }
+        
+        public static ByteStringContext<ByteStringMemoryCache>.InternalScope GetSliceFromId(ByteStringContext context, ReadOnlySpan<char> id, out Slice idSlice, byte? separator = null)
```

Comment:
Shouldn't it be called `GetLoweredIdSliceFromId()` and out parameter `lowerIdSlice` instead of `idSlice`?

Reply:
Sounds good

Reply:
done